app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -471,11 +479,25 @@ export default Component.extend(ComposerUploadUppy, {
   },
 
   _renderUnseenHashtags(preview) {
-    const unseen = linkSeenHashtags(preview);
+    let unseen;
+    const hashtagContext =
+      this.site.hashtag_context_configurations["topic-composer"];
```

Comment:
minor but wondering if we need the `_context_` here. `this.site.hashtag_configurations["topic-composer"]` looks descriptive enough to me